File: Python/List-2/sum67.py
def sum67(nums):
  
  total = 0
  found6 = False

  for i in range(len(nums)):
    if nums[i] == 6:
      found6 = True
    if not found6:
      total += nums[i]
    if nums[i] == 7 and found6:
        found6 = False

  return total

File: Java/Array-2/sum67.java
public int sum67(int[] nums) {
  int sum = 0;
  boolean stop = false;

  for (int i = 0; i < nums.length; i++) {
    if (nums[i] == 6)
      stop = true;
    if (stop == false)
      sum += nums[i];
    if (nums[i] == 7 && stop == true)
      stop = false;
  }
  return sum;
}
